### PR TITLE
Re-work technical qualification flow

### PR DESCRIPTION
**Pull request by ajaxscape**
https://eaflood.atlassian.net/browse/WE-911

Also prevent saving to a "Read only" entity such as Standard Rules

### CODE REVIEW

File: src/views/technicalQualification.html [hunk 1]
```diff
@@ -12,8 +12,13 @@
       {{> common/pageHeading pageHeading }}
 
       <p>
```
Comment (duncannixon) [inline src/views/technicalQualification.html]: Need to add IDs and add unit test

File: src/views/technicalQualification.html [hunk 2]
```diff
@@ -12,8 +12,13 @@
...
-        Your site must be managed by a 'competent person'. To prove this, you must provide evidence of a technical
-        competence qualification, for example from the Waste Management Industry Training and Advisory Board (WAMITAB).
+        To prove you are able to manage your operations correctly, you must upload evidence of technical competence
+        qualifications.
+      </p>
+      <p>
```
Comment (duncannixon) [inline src/views/technicalQualification.html]: And again

File: src/views/upload/technicalQualification/wamitabQualification.html [hunk 3]
```diff
@@ -20,10 +20,13 @@
       <div id="has-no-annotations" class="form-group">
         <div id="wamitab-qualification-description">
           <p>
-            Check the <a target="_blank" rel="noopener noreferrer" href="https://wamitab.org.uk/competence/operator-competence/">{{> abbreviations/WAMITAB }} risk tables (opens new tab)</a> to find out which qualification you need for this permit. If the qualification is over 2 years old you also need a continuing competence certificate.
+            <span class="bold-small">Upload copies of the certificates.</span>
```
Comment (duncannixon) [inline src/views/upload/technicalQualification/wamitabQualification.html]: ID and unit test needed

Comment (ajaxscape) [reply src/views/upload/technicalQualification/wamitabQualification.html]: The dynamic component of this which is whether the div with id="wamitab-qualification-description" exists or not is already covered in the "when there are annotations" and "when there are no annotations" tests in the uploadHelper.  An option of descrptionId = "wamitab-qualification-description" is passed in within the wamitabQualification.route.test.  I would argue that tests written to test all the hard coded text within the html within the page would render the tests brittle.

File: src/views/technicalQualification.html (hunk 1 above)
Comment (ajaxscape) [reply src/views/technicalQualification.html]: I have added an id here and test for it's existence however I would argue that tests written to test all the hard coded text within the html within the page is exactly as required would render the tests brittle.

**Pull request merged**